sdk/template-dpg/Azure.ServiceTemplate.Template/api/Azure.ServiceTemplate.Template.netstandard2.0.cs

```diff
@@ -1,4 +1,4 @@
-namespace Azure.Template.Generated
+namespace Azure.ServiceTemplate.Template
```

Comment:
Curious: if we're not created generated code, how do we have the API listing here?

Reply:
We can move this. and script will generate it.

Reply:
I placed a simple api file here for reference. Our script will update it to the right content.